### DISCUSSION TITLE
Integrate with libsecret library

### DISCUSSION THREAD

**Discussion by vanillajonathan** [NickvisionApps/Denaro#653] [Ideas]
Denaro lets you use a password for accounts.
Add integration with the libsecret library to integrate with Seahorse, aka "Password and Keys".

**Comment by fsobolev** [comment]
I'm against this. Making accounts unlock automatically by saving and retrieving passwords from libsecret means that if somebody has physical access to your device when it's not locked they can get access to your data in Denaro.

**Reply by vanillajonathan** [reply]
On the other hand, without libsecret users may be tempted to use weak passwords but when using libsecret users may be more likely to use strong passwords.

**Reply by nlogozzo** [reply]
> I'm against this. 

Me too

> On the other hand, without libsecret users may be tempted to use weak passwords but when using libsecret users may be more likely to use strong passwords.

I don't see the correlation...if your worried about users using weak passwords maybe we can have a Weak, Good, Strong indicator like when creating a user account password

**Reply by vanillajonathan** [reply]
Yeah, a password strength indicator is another good idea.

Maybe on Windows you could integrate with Windows Hello so users could authenticate with their fingerprint?

**Reply by nlogozzo** [reply]
> Yeah, a password indicator is another good idea.

Use https://github.com/NickvisionApps/Denaro/issues/656 to track

> Maybe on Windows you could integrate with Windows Hello so users could authenticate with their fingerprint?

We don't support windows anymore as of now... maybe revist this when we make an Avalonia UI version...but probably not...I'll see when I add secret support to Aura

**Reply by vanillajonathan** [reply]
KDE has KWalletManager, I don't know if it supports some master password or something.
